miso-web/src/main/webapp/scripts/indexDistance_ajax.js

```diff
@@ -36,14 +36,19 @@
         data: JSON.stringify(data)
       }).success(function(data) {
         var results = '';
-        if (!data || !data.length) {
+        if (data.warningEditDistance > data.errorEditDistance) {
+          results += 'Indices with edit distance of less than ' + Constants.warningEditDistance + ' are considered near-matches.\n';
```

Comment:
Should be "...distance of less than x+1" or "...distance of x or less". Same below

Reply:
I find the "edit distance" terminology to be confusing, and I'd like to reword it to "Indices with x or fewer mismatches" and change the page's "minimum edit distance" to "minimum mismatches". What do y'all think?

Reply:
I don't have a strong opinion either way. Just make sure the User Manual gets updated too

Reply:
Updated. PR: https://github.com/miso-lims/walkthroughs/pull/90

Reply:
That's good too, but I meant the User Manual which is in `/docs` of this repo

Reply:
Ah, I see you got that already too though :+1: